server/migrations/database.js

```diff
@@ -35,7 +35,7 @@ CREATE TABLE IF NOT EXISTS userDetails(
 const entries = `
 DROP TABLE IF EXISTS entries cascade;
```

Comment:
Do not drop the table first.

You could have a different migration meant to just drop your tables.

I imagine you have to add a new table to your schema and you run this and lose all your previous tables (and entries).

Reply:
oh now i get. I will work on it

Reply:
please check the below commit for update